pages/models.py

```diff
@@ -30,12 +31,206 @@
     InfoBoxBlock, \
     TestimonialBlock, \
     AllyLogoBlock, \
-    AssignableBookBlock
+    AssignableBookBlock, \
+    DividerBlock, \
+    APIRichTextBlock, \
+    CTAButtonBarBlock, \
+    CTALinkBlock
 
-from .custom_fields import \
-    Group
+from .custom_fields import Group
 import snippets.models as snippets
 
+# Constants for styling options on Root/Flex pages
+# consider moving to a constants.py file
+CARDS_STYLE_CHOICES = [
+    ('rounded', 'Rounded'),
+    ('square', 'Square'),
+]
+HERO_IMAGE_ALIGNMENT_CHOICES = [
+    ('left', 'Left'),
+    ('right', 'Right'),
+    ('topLeft', 'Top Left'),
+    ('topRight', 'Top Right'),
+    ('bottomLeft', 'Bottom Left'),
+    ('bottomRight', 'Bottom Right'),
+]
+HERO_IMAGE_SIZE_CHOICES = [
+    ('auto', 'Auto'),
+    ('contain', 'Contain'),
+    ('cover', 'Cover'),
+]
+SECTION_CONTENT_BLOCKS = [
+    ('cards_block', blocks.StructBlock([
+        ('cards', blocks.ListBlock(
+            blocks.StructBlock([
+                ('text', APIRichTextBlock()),
+                ('cta_block', blocks.ListBlock(CTALinkBlock(required=False, label="Link"),
+                    default=[],
+                    max_num=1,
+                    label='Call To Action'
+                )),
+            ]),
+        )),
+        ('config', blocks.StreamBlock([
+            ('card_size', blocks.IntegerBlock(min_value=0, help_text='Width multiplier. default 27.')),
+            ('card_style', blocks.ChoiceBlock(choices=CARDS_STYLE_CHOICES)),
+        ], block_counts={
+            'card_size': {'max_num': 1},
+            'card_style': {'max_num': 1},
+        }, required=False)),
+    ], label="Cards Block")),
+    ('text', APIRichTextBlock()),
+    ('html', blocks.RawHTMLBlock()),
+    ('cta_block', CTAButtonBarBlock()),
+]
+
+
+class LayoutSnippetSerializer(Field):
+    def to_representation(self, value):
+        return {
+            'layout': value.layout,
+            'background_image': value.background_image_api_response,
```

Comment:
this need to be removed?

Reply:
yes, thank you. i thought it was somewhere else.